README.md

```diff
@@ -17,7 +17,7 @@ The demo has a customer application that uses the pre-built customer chat widget
 #### Step 1: Create StartVideoCall Amazon Lex Bot
 This bot is needed for the contact flow (`Chime Connect Integration flow.json`). This bot helps in branching the Amazon Connect contact flow decision making when the customer wants to do a video enabled chat or just simple chat.
 
-1. Goto [Amazon Lex](https://console.aws.amazon.com/lex)
+1. Goto [Amazon Lex](https://console.aws.amazon.com/lex). Here, please use Lex V1 console if it takes you diretly to V2 console. You can go to V1 console by clicking 'Return to the V1 console'.
```

Comment:
Is V2 the default right now? Can we just say:
>Please use Lex V1 console. You can change to V1 console by clicking 'Return to the V1 console'.

Reply:
Atleast not for me, it was showing V1 itself, but I correct the sentence as suggested, thanks!

Reply:
Updated it thanks!